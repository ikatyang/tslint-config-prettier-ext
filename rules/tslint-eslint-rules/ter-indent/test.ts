if (true) {
console.log(123);
}
