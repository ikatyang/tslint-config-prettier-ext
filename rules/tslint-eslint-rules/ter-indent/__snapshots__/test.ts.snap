// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
if (true) {
console.log(123);
}

======
if (true) {
  console.log(123);
}

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (ter-indent) /rules/tslint-eslint-rules/ter-indent/test.ts[2, 1]: Expected indentation of 2 spaces but found 0.
"
`;
